package.json

```diff
@@ -60,5 +60,8 @@
     "bench": "pnpm run build:cjs && tsx benchmarks/index.ts",
     "prepare": "husky install",
     "publish:jsr": "jsr publish --dry-run"
+  },
+  "dependencies": {
+    "tshy": "^3.0.2"
```

Comment:
Was this intended to be a devDep? (it's the monorepo root so not harmful of course)

Reply:
Fixed! Thanks